recipe/meta.yaml

```
@@ -52,8 +52,6 @@ outputs:
     requirements:
       run:
```

We'll likely need some form of host-deps here to get the right run-exports per output (to be verified once CI has run through):
```suggestion
    requirements:
      host:
        - libboost-python-devel
        - libboost-devel
      run:
```

---

Actually, these currently get picked up correctly because the output is named the same as the feedstock. That has it's own kinds of problems though (like not running the test suite for `cctx-base` as expected), so I'll fix that.